packages/core/src/config/types.ts

```diff
@@ -51,23 +54,26 @@ export type StyleguideRawConfig<T = undefined> = {
   doNotResolveExamples?: boolean;
   recommendedFallback?: boolean;
 
-  rules?: RuleMap<BuiltInCommonRuleId | BuiltInCommonOASRuleId, RuleConfig, T>;
+  rules?: RuleMap<BuiltInCommonOASRuleId, RuleConfig, T>;
```

Comment:
I believe the `rules` section should contain the AsyncAPI rules as well:

```suggestion
  rules?: RuleMap<BuiltInCommonOASRuleId | BuiltInAsync2RuleId, RuleConfig, T>;
```

Reply:
Why do you think so?
This change will affect minimal, all, recomended rules and change the previous rule sets.

Reply:
Discussed offline - out of this scope.

Reply:
Oh, indeed. I'll create an issue to fix that separately.